tests/integration/base_block_test.go

```diff
@@ -221,6 +240,26 @@ func retryBlockImageCountCheck(helper *clients.TestClient, imageCount, expectedC
 	return false
 }
 
+func retryPVCheck(k8sh *utils.K8sHelper, name string, exists bool, status string) bool {
+	inc := 0
+	for inc < utils.RetryLoop {
+		pv, err := k8sh.GetPV(name)
+		if err != nil {
+			if !exists {
+				return true
+			}
+		}
+		if exists {
+			if fmt.Sprintf("%s", (*pv).Status.Phase) == status {
+				return true
+			}
+		}
+		time.Sleep(time.Second * utils.RetryInterval)
```

Comment:
in the integration tests, it's helpful to write to the log whenever we have a sleep so we can troubleshoot what even we're waiting for